content/ai/concepts/generative-ai/concepts/code/code.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Code'
+Description: 'Coding was one of the first areas commercialized with GenAI.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
+  - 'AI'
```

Comment:
Large Language Model and Transformers would be good tags, but they would also need to be added to the [Tags file](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) in this PR.